cli/tests/pcluster/config/test_update_policy.py

```diff
@@ -30,6 +30,7 @@
     "is_fleet_stopped, old_max, new_max, expected_result",
     [
         pytest.param(True, 10, 9, True, id="stopped fleet and new_max < old_max"),
+        pytest.param(False, "10", "9", False, id="stopped fleet and new_max < old_max"),
```

Comment:
Invert "is_fleet_stopped" value or "id". with `is_fleet_stopped`= true the fleet is running